admin/tests/integration/components/organizations/information-section_test.js

```diff
@@ -43,6 +43,32 @@ module('Integration | Component | organizations/information-section', function (
     assert.dom('.organization__canCollectProfiles').hasText('Oui');
   });
 
+  module('Displaying whether or not the items of this campaign will be exported in results', function () {
+    test("it should display 'Oui' when organizations is SUP or PRO", async function (assert) {
+      // given
+      const organization = EmberObject.create({ type: 'SUP', showSkills: true });
+      this.set('organization', organization);
+
+      // when
+      await render(hbs`<Organizations::InformationSection @organization={{this.organization}} />`);
+
+      // then
+      assert.dom('.organization__showSkills').hasText('Oui');
+    });
+
+    test("it should display 'Non' when organizations is SCO", async function (assert) {
```

Comment:
[suggestion] 
`it should display 'Oui' when showskills set to true`

`it should display 'Non' when showskills set to false`

le type n'est pas intéressant dans le test. il nous sert juste à être ISO avec le fonctionnel en production.

Reply:
yes.